docs/tutorials/prefill-concurrency.md

```
@@ -54,13 +54,11 @@ With --prefill-concurrency 3:
 
 Once a request receives its first token, it releases its prefill slot and moves to decode—allowing the next request to start prefilling.
 
-<Warning>
-Requires `--streaming` to be enabled. Without streaming, AIPerf can't detect when the first token arrives.
-</Warning>
+> [!WARNING]
+> Requires `--streaming` to be enabled. Without streaming, AIPerf can't detect when the first token arrives.
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)